examples/navigation/epics/fetchReposByUser.js

```diff
@@ -0,0 +1,36 @@
+import { merge } from 'rxjs/observable/merge';
+import { LOCATION_CHANGE } from 'react-router-redux';
+import { ajax } from 'rxjs/observable/dom/ajax';
+import * as ActionTypes from '../actionTypes';
+
+// Hack since react-router-redux doesn't hook
+// into react-router to grab parsed params.
+const USER_PATTERN = /repos\/([a-zA-Z0-9_]+)/;
+
+export default function fetchReposByUser(action$) {
+  const fetchIntents$ = action$.ofType(LOCATION_CHANGE)
+    .map(
+      ({ payload }) => payload.pathname.match(USER_PATTERN)
+    )
+    .filter(match => !!match)
+    .map(([, user]) => user);
+  return merge(
+    fetchIntents$.map(user => ({
+      type: ActionTypes.REQUESTED_USER_REPOS,
+      payload: {
+        user
+      }
+    })),
+    fetchIntents$
+      .switchMap(user =>
+        ajax.getJSON(`https://api.github.com/users/${user}/repos`)
+          .map(res => ({
+            type: ActionTypes.RECEIVED_USER_REPOS,
+            payload: {
+              repos: res,
+              user
+            }
+          }))
```

Comment:
@blesh I don't believe we want to advocate that for these sort of circumstances because it will confuse them and create bugs when they want to add error handling. e.g. if they put a `.catch()` that returns an error action inside the switchMap, now will wrongfully be mapped as the success response. If they place that `.catch()` outside the switchMap, it is too late and will prevent their Epic from listening to future actions.